custom_model_runner/setup.py

```diff
@@ -4,34 +4,49 @@
 This is proprietary source code of DataRobot, Inc. and its affiliates.
 Released under the terms of DataRobot Tool and Utility Agreement.
 """
+import importlib.util
+from pathlib import Path
 from setuptools import setup, find_packages
-import os
 
-from datarobot_drum.drum.description import version, project_name
-from datarobot_drum.drum.enum import SupportedFrameworks, extra_deps
+
+def direct_import(path):
```

Comment:
how did this work before?

Reply:
https://github.com/psf/requests/blob/15585909c3dd3014e4083961c8a404709450151c/setup.py#L78

Hmm maybe that is a little more elegant. Since I still have a failing test I may adapt our code.